### ISSUE TITLE
IntelliSense fails after backspace

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode-cpptools#131]
ms-vscode.cpptools-0.8.1
- take the code below
- run the steps as show in the gif
- ctrl+space -> see suggestions
- type `a` -> suggestions are narrowed down
- delete `a` -> 🐛 default suggestions show

![aug-05-2016 15-41-07](https://cloud.githubusercontent.com/assets/1794099/17439010/24d8f468-5b26-11e6-8201-9d41888abb47.gif)

```
// my first program in C++

int addition (int a, int b)
{
  int r;
  r=a+b;
  return r;
}

int main ()
{
  int z;
  z = addition (5,3);
}

```

**Comment by greazer** [comment]
This will be fixed in the next release of the extension. Thanks!

**Comment by sridmad** [comment]
This should be fixed in 0.9.0 version of the extension. Please update and let us know if the issue is resolved.
